### DISCUSSION TITLE
Installation

### DISCUSSION THREAD

**Discussion by alexeyKrIgo** [littlektframework/littlekt#218] [Q&A] [answered]
Hello,

I am new in creating kotlin projects with gradle. As startup I created a project without activities using androidstudio. The problem comes when I try to add the gradle.build.kts script by copying and paste the code that is indicated in the README.md because there are fields that are not recognized during the synchronization (like theones are inside the "kotlin" field). However I managed to install the dependencies just by adding and adapting this part of code:

```
val littleKtVersion = "0.8.0" // get the latest release at the top
val kotlinCoroutinesVersion = "1.6.4" // or whatever version you are using
dependencies {
            implementation("com.lehaine.littlekt:core:$littleKtVersion")
            implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:$kotlinCoroutinesVersion")
}
```

Is it ok or should I read some documentation that I don't know about?

**Comment by LeHaine** [comment, marked as answer]
Hello!

The example in the README is meant to show the snippet of code that can be used to add to your existing Gradle build script. If you managed to get it working then you should be good. But if you need an example check out the following repository as it has the bare minimum needed to run a LittleKt project: https://github.com/littlektframework/littlekt-game-template
